### PR TITLE
9 delete comment

### PR DESCRIPTION
**Pull request by manj-93**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -380,3 +380,32 @@ describe("PATCH /api/articles/:article_id", () => {
         });
     });
 });
+
+describe('DELETE /api/comments/:comment_id', () => {
+    test('204: successfully deletes a comment', () => {
+        return request(app)
+        .delete('/api/comments/1')
+        .expect(204)
+        .then((result)=>{
```
Comment (nc-mark-t) [inline __tests__/app.test.js:389]: You don't need to test this because a 204 is no content so none is sent

**Review by nc-mark-t — COMMENTED** [review]
Really neat 👍
